internal/csi-addons/rbd/volumegroup.go

```diff
@@ -527,6 +597,21 @@ func (vs *VolumeGroupServer) ModifyVolumeGroupMembership(
 		}
 	}
 
+	// Enable mirroring after modification of volume group
```

Comment:
i think the check need to be changed to `!` do enable mirroring if its disabled.